### PR TITLE
fix: use attachTo in failing tests with jsdom v21.1

### PR DESCRIPTION
**Pull request by cexbrayat**
`attachTo` can be removed when https://github.com/jsdom/jsdom/issues/3502 is fixed
 With jsdom v21.1, `getComputedStyle` is cached and not updated for elements not in the DOM
 so some our tests fail (because the elements are checked a first time)

We'll probably see issues in Vue projects using VTU and jsdom as well, and we can point to this PR if that happens

### CODE REVIEW

File: tests/isVisible.spec.ts
Comment (lmiller1990) [inline tests/isVisible.spec.ts:56]: Nice. I almost wonder if attachTo: document.body should be the default behavior. I wonder if that'd be a breaking change?

**Review by lmiller1990 — APPROVED** [review]
Fine either way, but any reason we need to update to this version of jsdom asap? Should we just wait for the release w/ the patch?

**Comment by cexbrayat** [comment]
@lmiller1990 We don't have to, I'm fine either way 😉

It's just that:
- I was happy to have a workaround and to share it with you
- I'm not even sure jsdom is going to fix it
- I wanted to have something to point out if someone opens an issue (we had the same issues in our projects of course)
- It unblocks Renovate

File: tests/isVisible.spec.ts
Comment (cexbrayat) [reply tests/isVisible.spec.ts:56]: Maybe we should, I don't know what are the downsides (most testing libs in other ecosystems do attach components right?). As you have more context, do you know why this is not the case in VTU?

**Comment by lmiller1990** [comment]
That's fair - happy to move forward with this.

Once we merge up all the deps PRs, we can do another release - good to have the latest code in the wild, even if it's just dev dependencies updates that don't make a difference - good to keep a fast release cadence.

**Pull request merged**
